experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/OTLPMetricExporterBase.ts

```diff
@@ -104,6 +106,16 @@ function chooseTemporalitySelector(
   return chooseTemporalitySelectorFromEnvironment();
 }
 
+function chooseAggregationSelector(
+  config: OTLPMetricExporterOptions | undefined
+) {
+  if (config?.aggregationSelector) {
+    return config.aggregationSelector;
+  } else {
+    return (_instrumentType: any) => Aggregation.Default();
```

Comment:
I'm thinking on whether this line is correct. 

Previously aggregationSelector was `undefined` for this scenario instead.

Reply:
But I'm thinking DEFAULT_AGGREGATION_SELECTOR here is probably still more correct than `undefined.bind()` [here](https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/sdk-metrics/src/export/PeriodicExportingMetricReader.ts#L65)?

Reply:
I reverted to `return (_instrumentType: any) => Aggregation.Default();` since i think with `DEFAULT_AGGREGATION_SELECTOR ` browserTests failed? 🤔

Or maybe they will fail again :)

Reply:
> I reverted to `return (_instrumentType: any) => Aggregation.Default();`

I prefer this solution :+1: